pkg/updater/pubsub.go

```diff
@@ -195,10 +195,10 @@ func processGCSNotifications(ctx context.Context, log logrus.FieldLogger, q *con
 }
 
 var namedDurations = map[string]time.Duration{
-	"podinfo.json":  time.Second,     // Done
-	"finished.json": time.Minute,     // Container done, wait for prowjob to finish
-	"metadata.json": 5 * time.Minute, // Should finish soon
-	"started.json":  time.Second,     // Running
+	"podinfo.json":  30 * time.Second, // Done
```

Comment:
why changing these values?

Reply:
> Also allow a larger buffer between when we receive a pubsub notification and when reprocess the tab.

Reply:
Suspect we may get the pubsub notification before it appears in the `ls`